docs/source/concepts/DynamicalStructureFactorFromAbInitio.rst

```diff
@@ -0,0 +1,163 @@
+form of dynamical structure factor  is recorded. In principle, one obtains phonon (vibrational) spectrum which is often
+quite  difficult to analyse. In order to better understand experimental results one can use modeling methods. One of the
+most successful theoretical method in predicting phonon spectrum is ab-initio method DFT [1].
+
+.. image:: ../images/dft_phonon_scheme.png
+    :align: center
+
+Usual approach in calculating phonon spectra within DFT is presented in the figure above. First, one defines initial
+guess for the structure in interest. The initial guess should be as close as possible to an experimental structure.
+Then, one optimise structure within DFT. By optimising structure one should understand finding the closest to the
+initial guess local minimum of energy within accuracy of DFT method. At the closest minimum there
+should be no effective force on atoms. For that structure the dynamical matrix, e.g, second derivative of system Hamiltonian with respect to
+atomic displacements is calculated. Eigenvectors obtained from diagonalisation of this matrix are atomic displacements
+and eigenvalues are squared frequencies of vibrations. Those vibrations are sometime called *modes* or *fundamentals*.
+Using atomic displacements and frequencies obtained by DFT method one can calculate theoretical :math:`S(\mathbf{Q}, \omega)`. :math:`S(\mathbf{Q}, \omega)`
+is calculated for each atom separately. Total S is sum over all partial atomic contributions.
+
+
+Working equations
++++++++++++++++++
+
+Powder
+~~~~~~
+
+.. image:: ../images/s_powder_scheme.png
+    :align: center
+
+From DFT calculations one usually obtains phonon data for infinite single crystal (there are ways to perform
+calculations for molecule, e.g, gas phase). In order to compare it with experiment in which sample is in the form of
+powder one has to perform powder averaging. Usually semi-empirical formula is used [2], [3]:
+
+:math:`S^j (\mathbf{Q},\omega_i) = S^j (Q,\omega_i) = \frac{Q^2(\omega_i) TrB_{\omega_i}}{3} exp\left(-Q^2(\omega_i) \alpha^j_{\omega_i} coth^2\left(\frac{\hbar \omega_i}{2 k_B T}\right)  \right)\sigma^j`
+
+where B and A are tensors created from atomic displacements in the following way:
+
+:math:`B^j_{\omega_i} = c^j_{\omega_i}(c^{j}_{\omega_i})^T M^j  \frac{C_1}{\omega_i}`
+
+:math:`A^j = \sum_i B^j_{\omega_i}`
+
+with
+
+:math:`Q` -- momentum transfer which occurs during neutron scattering. In principle, momentum transfer :math:`\mathbf{Q}` is a vector but due to powder averaging S for the powder case depends on length of :math:`\mathbf{Q}`
+
+:math:`\alpha^j_{\omega_i}` -- semi-empirical parameter calculated as: :math:`\alpha^j_{\omega_i} = \frac{1}{5} \left \lbrace Tr A^j  + \frac{2 B^j_{\omega_i}: A^j}{Tr B^j_{\omega_i}} \right\rbrace`
+
+:math:`Tr` -- trace operation
+
+:math:`:` --  tensor contraction operation
+
+:math:`j` -- indicates j-th atoms
```

Comment:
use math mode for `j` in the description. Same for `i` in the next line, and so on